engine/utils/String.cpp

```diff
@@ -283,6 +285,16 @@ String String::FromLong(const long& value)
     return GetFromFormat("%ld", value);
 }
 
+String String::FromSizeT(const size_t& value)
+{
+    return GetFromFormat("%lu", value);
+}
+
+Hash::StringId String::WSID(const String& str)
+{
+    return Hash::WSID(str.Str());
+}
+
```

Comment:
Avoid adding this dependency when the free floating function for `char*` exists and works just fine.